web/src/components/TickerTable/index.tsx

```diff
@@ -9,14 +9,18 @@ interface Props {
     markets: Market[];
     redirectToTrading: (key: string) => void;
     setCurrentBidUnit: (key: string) => void;
+    isFixedPrecision: boolean
```

Comment:
please remove this prop, there is no case when we need Market table without fixed precision.